Platform/Bosc/XiangshanSeriesPkg/NanhuDev/NanhuDev.dsc

```diff
@@ -429,11 +448,13 @@
   MdeModulePkg/Universal/SecurityStubDxe/SecurityStubDxe.inf
 !endif
 
-  UefiCpuPkg/CpuIo2Dxe/CpuIo2Dxe.inf
+  Silicon/RISC-V/ProcessorPkg/Universal/PciCpuIo2Dxe/PciCpuIo2Dxe.inf
```

Comment:
Any reason not to use edk2 drivers like UefiCpuPkg/CpuMmio2Dxe?

Reply:
Because I noticed that there is a driver in directory "Silicon/RISC-V/ProcessorPkg/Universal/PciCpuIo2Dxe/", it actually looks no different from directory " edk2 drivers like UefiCpuPkg/CpuMmio2Dxe"

Reply:
Initially, this module was duplicated in different architectures (originally in ARM). But recently it is added as UefiCpuPkg/CpuMmio2Dxe which can be used by multiple architectures. So, please use UefiCpuPkg/CpuMmio2Dxe itself. That helps when this RISC-V specific PciCpuIo2Dxe gets removed.

Reply:
Okay ， Got it